src/adapter/pg.cr

```diff
@@ -141,7 +141,7 @@ class Granite::Adapter::Pg < Granite::Adapter::Base
     log statement, elapsed_time, value
   end
 
-  def ensure_clause_template(clause : String) : String
+  protected def ensure_clause_template(clause : String) : String
```

Comment:
👍 seems like a good visibility change. I feel like the name of this method is totally opaque, but I can't think of a better name right now.